python/benchmarks/common.py

```diff
@@ -9,7 +9,7 @@
 import numpy as np
 
 
-def generate_pseudo_random_dataframe(n, freq="S", end_timestamp="1/1/2023"):
+def generate_pseudo_random_dataframe(n, freq="s", end_timestamp="1/1/2023"):
```

Comment:
What does changing "S" to "s" do?

Reply:
"S" is apparently deprecated, and we should use "s"